src/applications/mhv/medications/util/constants.js

```diff
@@ -30,40 +30,109 @@ export const dispStatusForRefillsLeft = [
-We may use this status for either of these reasons:
-* We’re not sure when you’ll need to fill this prescription, or
-* You told us you have too much of this medication or supply
-If you need this prescription now, you can request to fill or refill it.`,
-
-  hold: `We put a hold on this prescription. You can’t request a refill until we remove the hold.
-We may use this status for either of these reasons:
-* You told us you have too much of this medication or supply, or
-* There’s a problem with this prescription
-If you need this prescription now, call your VA pharmacy.`,
-
-  refillinprocess: `We’re processing a fill or refill for this prescription. We’ll update the status here when we ship your prescription.`,
-
-  discontinued: `You can’t refill this prescription. We may use this status for either of these reasons:
-* Your provider stopped prescribing this medication or supply to you, or
-* You have a new prescription for the same medication or supply
-If you have questions or need a new prescription, send a message to your care team.`,
-
-  submitted: `We got your request to fill or refill this prescription. We’ll update the status when we process your request.
-Check back for updates. If we don’t update the status within 3 days after your request, call your VA pharmacy.`,
-
-  expired: `This prescription is too old to refill.
-An expired prescription doesn’t mean the medication itself is expired. Check the prescription label for the expiration date of the medication.
-If you need more of this prescription, request a renewal.`,
-
-  transferred: `We moved this prescription to our My VA Health portal.`,
+  active: [
+    {
+      value: `This is a current prescription. If you have refills left, you can request a refill now.`,
+    },
+    {
+      continued: true,
+      value: `Note: `,
+      weight: 'bold',
+    },
+    {
+      value: `If you have no refills left, you’ll need to request a renewal instead.`,
+    },
+  ],
+  activeParked: [
+    {
+      value: `Your VA provider prescribed this medication or supply to you. But we won’t send any shipments until you request to fill or refill it.`,
+    },
+    {
+      value: `We may use this status for either of these reasons:`,
+    },
+    {
+      value: [
+        `We’re not sure when you’ll need to fill this prescription, or`,
+        `You told us you have too much of this medication or supply`,
+      ],
+    },
+    {
+      value: `If you need this prescription now, you can request to fill or refill it.`,
+    },
+  ],
+  hold: [
+    {
+      value: `We put a hold on this prescription. You can’t request a refill until we remove the hold.`,
+    },
+    {
+      value: `We may use this status for either of these reasons:`,
+    },
+    {
+      value: [
+        `You told us you have too much of this medication or supply, or`,
+        `There’s a problem with this prescription`,
```

Comment:
@vmenshutin-bylight I'm assuming we don't need a period here?

Reply:
@agravell047 there was none before, it is a bulleted list